.github/workflows/build_documentation.yml

```diff
@@ -14,6 +14,6 @@ jobs:
       package: course
       path_to_docs: course/chapters/
       additional_args: --not_python_module
-      languages: en es th
+      languages: en es th tr
```

Comment:
Just FYI I added Turkish to the doc build so we can check it renders on the website :)